src/main/java/hng_java_boilerplate/user/dto/response/Response.java

```diff
@@ -0,0 +1,16 @@
+package hng_java_boilerplate.user.dto.response;
+
+import lombok.*;
+
+@Builder
+@AllArgsConstructor
+@RequiredArgsConstructor
+@Setter
+@Getter
+public class Response<T> {
+
+    private String status;
```

Comment:
This should be status_code, alternatively you can modify the ApiResponse to be generic and use that instead of having 2 of them doing the same thing